docs/design/_template.adoc

```diff
@@ -1,206 +0,0 @@
-= FeatureTemplate Design Doc
-// Author field:
-Jonh Smith <email@email.org>
-v0.1, 2020-01-01
-:draft:
-:toc:
-
-== Overview
-////
-:required:
-
-In a few sentences, describe the key objectives. Why do we need this feature?
-What are we trying to accomplish?
-
-Just a few sentences.
-////
-
-== Background
-////
-:required:
-
-Include as much information as necessary here to understand the design. Include
-glossary if necessary in this section. Links to examples, related projects
-or other design docs. Any previous/current version of this feature.
-
-Do not write ideas about how to solve the problem here.
-////
-
-=== Problem Statement
-////
-:required:
-
-State the problem this design solves, in a concise way. A few sentences, not more.
-////
-
-=== Requirements
-////
-:optional:
-
-State which requirements are necessary to consider this problem solved. This is in
-relation to the solution, not the problem.
-////
-
-== Expected User/Developer Experience
-////
-:required: Either User and/or Developer experience should be explained.
-
-Describe what
-////
-
-== Prior Art
-////
-:optional: But recommended.
-
-Link to other products available as comparatives to this design. For example,
-if another tool has a similar feature, list pros/cons/shortcomings of that tool.
-////
-
-== Detailed Design
-////
-:required:
-
-Full design on how to implement the feature. Use this section to also describe
-why some simpler approaches did not work. Mention other things to watch out
-for during implementation.
-
-Keep in mind that you might not be the person implementing the design.
-////
-
-=== Considered Solutions
-////
-:required:
-
-What solutions were considered, with a list of pros and cons of each solutions.
-////
-
-=== Recommended Solution
-////
-:required:
-
-What solution from the above are you recommending, and most importantly, WHY?
-////
-
-=== Public API
-////
-:optional: Required if there is any public API changes
-
-List any new or current API changes. List traits, methods, arguments and any
-types. A good way is to paste an example of the API in the language it will be
-implemented, for example (with Rust):
-
-[source,rust]
-----
-/// Confabulate the splines using reverse polarity. Can return an error if
-/// the space is asynchronously stochastic.
-pub fn confabulate(spline: &mut [&Spline], polarity: bool) -> Result<(), Error> {}
-----
-
-It's important to avoid implementations here and speak in general terms.
-////
-
-=== Prototype
-////
-:optional:
-
-If a proof of concept is available, include a link to the files here (even if
-it's in the same PR).
-////
-
-=== Security Considerations
-////
-:optional:
-
-How will this feature impact security, and what needs to be done to keep it
```

Comment:
None of this made it to the new document

Reply:
I think it's safe to delete `docs/design`. Any parts from there should be I think either moved to official IC docs, or to sdk's repo github wiki. Wdyt?